### PR TITLE
Fix spelling and lint errors

### PR DESCRIPTION
**Pull request by SoManyHs**
To appease the report card gods:
https://goreportcard.com/report/github.com/aws/amazon-ecs-cli

### CODE REVIEW

**Comment by SoManyHs** [comment]
Note: normally, the `go vet` and `go cyclo` sections are more important, but the errors reported from go vet are mostly frivolous (from tests) and the major offenses reported by go cyclo require more serious refactoring of major commands (e.g. `createCluster`), which I partially am not sure how to address due to the way go makes you constantly handle errors.

**Comment by aaithal** [comment]
@SoManyHs you can exclude mocks and test files from `go vet` and `go cyclo`. [Here's](https://github.com/aws/amazon-ecs-agent/blob/dev/Makefile#L171) an example for the same.

File: ecs-cli/modules/clients/aws/cloudformation/client.go [hunk 1]
```diff
@@ -238,19 +238,18 @@ func (c *cloudformationClient) waitUntilComplete(stackName string, hasFailed fai
...
-				log.Debug("Stack creation failed. Getting first failed event")
-				if failureEvent, err := c.firstStackEventWithFailure(stackName, nil, failureStates); err == nil {
-					log.WithFields(log.Fields{
-						"reason":       aws.StringValue(failureEvent.ResourceStatusReason),
-						"resourceType": aws.StringValue(failureEvent.ResourceType),
-					}).Error("Failure event")
-				}
-				return fmt.Errorf("Cloudformation failure waiting for '%s'. State is '%s'", successState, status)
+		_, exists := failureStates[status]
+		if exists {
```
Comment (PettitWesley) [inline ecs-cli/modules/clients/aws/cloudformation/client.go]: I propose `exists` => `failure`. I feel like `if failure` would read more cleanly.

File: ecs-cli/modules/config/readwriter_v1.go [hunk 2]
```diff
@@ -172,7 +172,7 @@ func (rdwr *YAMLReadWriter) Get(clusterConfig string, profileConfig string) (*CL
 	profilePath := credentialsFilePath(rdwr.destination)
 	configPath := ConfigFilePath(rdwr.destination)
 
-	// try to readINI first; it is either sucessful or it
```
Comment (PettitWesley) [inline ecs-cli/modules/config/readwriter_v1.go]: My inability to spell is disturbing...

File: ecs-cli/modules/clients/aws/cloudformation/client.go (hunk 1 above)
Comment (SoManyHs) [reply ecs-cli/modules/clients/aws/cloudformation/client.go]: that's fair, but I was strictly addressing things that came up through the tool. i can change it though.

Comment (SoManyHs) [reply ecs-cli/modules/clients/aws/cloudformation/client.go]: Actually looking closer at the code, `exists`makes more sense, so I'll keep it as is.

**Pull request merged**